ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -239,7 +239,8 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
         I18n.t("serviceDetail.onUpdateEnabledChannelsFailure"),
         "danger"
       );
-
+      // Reload profile
```

Comment:
```suggestion
      // it could happen that profile update fails due to version number mismatch
      // app has a different version of profile compared to that one owned by the backend
      // so we force profile reloading
```